packages/firebase_auth/firebase_auth/android/src/main/java/io/flutter/plugins/firebase/auth/AuthStateChannelStreamHandler.java

```diff
@@ -45,7 +43,7 @@ public void onListen(Object arguments, EventSink events) {
           if (user == null) {
             event.put(Constants.USER, null);
           } else {
-            event.put(Constants.USER, parseFirebaseUser(user));
+            event.put(Constants.USER, PigeonParser.parseFirebaseUser(user).toList());
```

Comment:
Was this a list before?

Reply:
here we are using an EventChannel which is not directly supported by Pigeon. Using the toList we are able to still format the FirebaseUser as a normal Pigeon Object and parse it back into an Object in Dart using PigeonFirebaseUser.decode

Reply:
I see, so in Dart it now requires a list rather than a Map as before?

Reply:
It's just that we use Pigeon to Parse the object now, the object is typed in Dart instead of just a map now